blog/meet-content-wro-marzec-2024-relacja/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'MeetContentWRO marzec 2024 - Relacja'
+authors: mrozek
+date: '2024-04-04'
+tags:
+  - 'meetup'
+  - 'nlu'
+  - 'smart search results'
+coverImage: '26MeetContentWRO.png'
+---
+
+20 marca MeetContentWRO ponownie gościł w inspirującym wnętrzu przy Szybowcowej 2, czyli w Nokia Garage. 
+
+Prezentację na temat: *Smart search results with Natural Language Understanding (NLU)* dostarczyło wspólnie trzech prelegentów.
+
+<!--truncate-->
+
+Nasi marcowi prelegenci to: 
+
+* Magdalena Jałówko, obecnie Product Owner w Nokii
+* [Adam Kąkol](https://www.linkedin.com/in/adam-kakol-14753b135/), który w Nokii jest Managerem obszaru AI-Driven Knowledge Solutions
+* [Natalia Rusin](https://www.linkedin.com/in/natalia-rusin-237930221/), która jest UX/UI Designerem  w Nokii
+
+## Język naturalny - cel i perspektywa
+
+Nokia zajmuje się tworzeniem urządzeń do komunikacji, na przykład różnych urządzeń radiowych czy nadajników. W związku z tym potrzebuje udokumentować swoje produkty w sposób ustandaryzowany i umożliwiający tysiącom pracowników, klientów i użytkowników rozeznanie się w ogromie sprzętu.
+
+Skrót NLU, czyli Natural Language Understanding, kryje w sobie obszar sztucznej inteligencji odpowiedzialny za interakcję między człowiekiem a komputerem, w szczególności za rozumienie i przetwarzanie języka naturalnego.
+
+Nic dziwnego, że potrzebne było zaangażowanie nie-ludzkich 😉 zasobów, żeby to wszystko ogarnąć i uczynić ‘przyjaznym’ dla użytkownika. 
+
+Zespół Adama określił cel i przyjął pewne założenia. 
+
+Określono dwie istotne perspektywy rozumienia języka naturalnego:
+
+* Użytkownika
+* Twórcy treści technicznych
+
+Cel z kolei był jasny: Poprawa wyników wyszukiwania w portalu dokumentacyjnym.
+
+## Intencje
+
+Za każdym pytaniem (search query) czai się jakaś potrzeba użytkownika, tak zwana intencja. 
+
+Żeby te intencje zidentyfikować, zespół Adama:
```

Comment:
W poprzednim zdaniu jest mowa o jednej intencji, więc wg mnie lepiej będzie to kontynuować w tym zdaniu.
```suggestion
Żeby tę intencję zidentyfikować, zespół Adama:
```